output/json/json.go

```diff
@@ -0,0 +1,152 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2021 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package json
+
+import (
+	"compress/gzip"
+	stdlibjson "encoding/json"
```

Comment:
I think this isn't necessary, but I am fine with it being that way ;)

Reply:
I had some issues when I imported it in `cmd/`, though that might have been a gopls issue instead of a Go one :man_shrugging: